Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/PermissionUtils.java

```diff
@@ -0,0 +1,45 @@
+package in.juspay.mobility.app;
+
+import static androidx.core.app.ActivityCompat.requestPermissions;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.webkit.JavascriptInterface;
+
+import androidx.core.app.ActivityCompat;
+
+public class PermissionUtils {
+
+    private static final int GENERIC_REQUEST_CODE = 100;
+
+    public interface PermissionCallback {
+        void onPermissionsGranted();
+        void onPermissionsDenied();
+    }
+
+    public static boolean checkIfAllPermissionsGranted(Context context, String[] requests) {
```

Comment:
can you write a comment above this function with a sample imputs this can take? Also why are we sending requests as an array of string?